Packs/Core/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Core - Investigation and Response",
+    "description": "Automates incident response",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": ["Endpoint", "Data Enrichment & Threat Intelligence"],
+    "tags": [],
+    "useCases": ["Malware"],
+    "keywords": [],
+    "marketplaces": [
+        "xsoar",
```

Comment:
this will be need to be removed before merge, but not during development.

Reply:
Will change to:
"marketplaces": [
        "marketplacev2"
    ]